.eslintrc

```diff
@@ -1,27 +1,20 @@
 {
   "parser": "babel-eslint",
-  "extends": "airbnb",
+  "extends": ["airbnb", "prettier"],
```

Comment:
I think that we should stop use airbnb's preset

Reply:
What are your reasons to remove the `airbnb` preset?
I think that this is out of scope of the PR

Reply:
Prettier only covers stylistic issues, we'll still need eslint and a good preset for non-style issues such as unused vars.